src/arm/ffi.c

```diff
@@ -566,8 +566,13 @@ ffi_prep_closure_loc (ffi_closure * closure,
   config[1] = closure_func;
 #else
   memcpy (closure->tramp, ffi_arm_trampoline, 8);
+#if defined (__QNX__)
+  msync(closure->tramp, 8, 0x1000000);	/* clear data map */
+  msync(codeloc, 8, 0x1000000);	/* clear insn map */
```

Comment:
@tromey Ooh sorry, I missed that. Regarding the constants I hope @s1341 (the author of this patch) can chime in on that.